# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: extensions/permissions/moz.build
# Commit: 44e715808901
# Full Hash: 44e71580890195d163912b96f41ef09e32570ec8
# Author: Tim Huang <tihuang@mozilla.com>
# Date: 2020-06-13 03:21:52
# Regressor Bug: 1587743
# File Overlap Count: 1
# Description:
#   Bug 1587743 - Part 1: Pre-compute the delegated permissions for the top-level content and store it in the WindowContext. r=baku,nika
#   
#   In order to delegate the permission to the top-level window, in this
#   patch, we pre-compute the permissions of the top-level context and set
#   them to the top-level WindowContext. So, the cross-origin iframe can
# ==============================================================================

diff -r a890be633ec3 -r 44e715808901 extensions/permissions/moz.build
--- a/extensions/permissions/moz.build	Fri Jun 12 16:18:15 2020 +0000
+++ b/extensions/permissions/moz.build	Fri Jun 12 16:31:49 2020 +0000
@@ -13,6 +13,7 @@
 EXPORTS.mozilla += [
     'Permission.h',
     'PermissionDelegateHandler.h',
+    'PermissionDelegateIPCUtils.h',
     'PermissionManager.h',
 ]
 
